CMakeLists.txt

```diff
@@ -68,15 +69,20 @@ endif(CUDA_FOUND)
 add_executable(gproshan gproshan.cpp)
 add_executable(test_geodesics test_geodesics.cpp)
 add_executable(test_image_denoising test_image_denoising.cpp)
+add_executable(test_mesh_denoising test_mesh_denoising.cpp)
```

Comment:
could be good have test folder with these codes, what do you think? or are they samples?